en/contributing/documentation.md

```diff
@@ -51,18 +51,22 @@ For setup information see: [Gitbook toolchain](https://toolchain.gitbook.com/set
 
 ## API Reference
 
-The C++ source code is annotated using comments using [Doxygen](https://www.stack.nl/~dimitri/doxygen/manual/index.html) syntax. You can extract the documentation on macOS or Linux when you [build the library](../contributing/build.md) using the command: 
+The C++ source code is annotated using comments using [Doxygen](https://www.stack.nl/~dimitri/doxygen/manual/index.html) syntax. You can extract the documentation to markdown files (one per class) when you [build the library](../contributing/build.md).
```

Comment:
@julianoes This is just a clarification/improvement of how our API Ref is generated. You don't need to review this doc.